src/stylesheets/notifications.less

```diff
@@ -1,42 +1,62 @@
-.sk-notifications {
+.sk-notification-container {
+    box-shadow: 0 1px 3px rgba(0, 0, 0, .1);
     position: absolute;
     z-index: 1;
 
-    overflow: hidden;
+    .sk-notifications {
```

Comment:
nit, this file ought to be called `notification.less` so it matches the transitionName.
And this selector should be `.sk-notifications`